input_test.go

```diff
@@ -69,25 +69,27 @@ func TestMouseClickNode(t *testing.T) {
 	t.Parallel()
 
 	tests := []struct {
+		file     string
 		sel, exp string
 		opt      MouseOption
 		by       QueryOption
 	}{
-		{`button2`, "foo", ButtonType(input.ButtonNone), ByID},
-		{`button2`, "bar", ButtonType(input.ButtonLeft), ByID},
-		{`button2`, "bar-middle", ButtonType(input.ButtonMiddle), ByID},
-		{`input3`, "foo", ButtonModifiers(input.ModifierNone), ByID},
-		{`input3`, "bar-right", ButtonType(input.ButtonRight), ByID},
-		{`input3`, "bar-right", Button("right"), ByID},
-		{`document.querySelector('#input3')`, "bar-right", ButtonType(input.ButtonRight), ByJSPath},
+		{"input.html", `button2`, "foo", ButtonType(input.ButtonNone), ByID},
+		{"input.html", `button2`, "bar", ButtonType(input.ButtonLeft), ByID},
+		{"input.html", `button2`, "bar-middle", ButtonType(input.ButtonMiddle), ByID},
+		{"input.html", `input3`, "foo", ButtonModifiers(input.ModifierNone), ByID},
+		{"input.html", `input3`, "bar-right", ButtonType(input.ButtonRight), ByID},
+		{"input.html", `input3`, "bar-right", Button("right"), ByID},
+		{"input.html", `document.querySelector('#input3')`, "bar-right", ButtonType(input.ButtonRight), ByJSPath},
+		{"click.html", `link`, "clicked", ButtonType(input.ButtonLeft), ByID},
```

Comment:
why not add the extra HTML to `input.html`? After all, this is `input_test.go`, and it will keep the test more consistent.

Reply:
@mvdan changed to input.html.